system-x/services/splunk/src/main/resources/splunk-operator-namespace.yaml

```diff
@@ -415,25 +435,10 @@ subjects:
   name: splunk-operator-controller-manager
   namespace: DESIRED_NAMESPACE
 ---
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  labels:
-    name: splunk-operator
-  name: splunk-operator-proxy-rolebinding
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: splunk-operator-proxy-role
-subjects:
-- kind: ServiceAccount
-  name: splunk-operator-controller-manager
-  namespace: DESIRED_NAMESPACE
----
 apiVersion: v1
 data:
   OPERATOR_NAME: '"splunk-operator"'
-  RELATED_IMAGE_SPLUNK_ENTERPRISE: SPLUNK_IMAGE
+  RELATED_IMAGE_SPLUNK_ENTERPRISE: quay.io/fuse_qe/splunk:9.0.3-a2
```

Comment:
this won't be replaced when you change the image in `defaultImage` method

Reply:
Yes, I removed that replacement. But If we want to have it only in defaultImage method, I will put it back.